indico/modules/designer/models/templates.py

```diff
@@ -69,8 +70,12 @@ class DesignerTemplate(db.Model):
         db.ForeignKey('indico.designer_image_files.id'),
         index=False,
         nullable=True
+      )
+    backside_template_id = db.Column(
+        db.ForeignKey('indico.designer_templates.id'),
+        index=False,
```

Comment:
I think it'd be a good idea to have an index on it so we can efficiently check whether a template is the backside of something.